src/_includes/header2.html

```diff
@@ -0,0 +1,53 @@
+{%- include google-tag-manager-body.html -%}
+
+<header class="siteHeader2">
+  <div class="siteHeader2-logoAndMenuButton">
+    {% if page.layout == 'landing' %}
+    <!-- For the "Guide To App Devs" - /welcome_to_tilt.html -->
+    <a class="imageLink imageLink--logo-imagemark" href="{{site.landingurl}}/">
+      {% svg assets/svg/logo-imagemark.svg %}
+    </a>
+    {% elsif page.layout == 'home' %}
+    {% svg assets/svg/Tilt-logo.svg class="siteHeader2-logoLink-svg" %}
```

Comment:
for my understanding, is there a reason you created a new svg logo? we already have 3 logo-wordmark.svg's - one with an alpha channel, one with solid colors, and one for animations

Reply:
Haha, this was kind of lazy of me. The new layout has a larger logo. Since default height and width are part of the SVG, I preferred to export a new SVG instead of using CSS to alter the existing dimensions. (Which would have meant hardcoding a seemingly arbitrary width in CSS.)

If we go with this new header, we also won't need the solid vs anim versions, since I removed the animation effect. (It was cute but I thought a bit distracting.) I also think people sometimes download logos for use from websites, so I named it "Tilt-logo" so it's a bit more of a branded product.

Overall, my assumption is that we'll make a decision whether to go with the new layout or revert to the existing layout, so I wanted to make it easier to delete one set or another rather than re-use existing elements.